BlazarBase/src/main/java/com/hubspot/blazar/base/BuildCommand.java

```diff
@@ -0,0 +1,61 @@
+package com.hubspot.blazar.base;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.Objects;
+import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.Lists;
+
+public class BuildCommand {
+  private static final String DEFAULT_CMD = "/bin/bash";
+  private static final String DEFAULT_ARG = "-c";
```

Comment:
Should the -c be part of the default command? Otherwise people will need to add it as the first arg which seems weird, ie:

```yaml
args:
  - -c
  - mvn deploy
```

Also it seems weird that the command to run is called `args` but I'm not sure what a better name would be